src/main/java/com/tlswe/awsmock/common/util/Constants.java

```diff
@@ -87,4 +87,70 @@
      * Property name for min boot time of a mock EC2 instance in seconds.
      */
     String PROP_NAME_INSTANCE_MIN_BOOT_TIME_SECONDS = "instance.min.boot.time.seconds";
+
+    /**
+     * Property name for vpc id.
+     */
+    String PROP_NAME_VPC_ID = "vpcId";
```

Comment:
Could you name these new properties as "network.vpcId", "network.vpcState" and so on? So that they seem more classified in the property file.